ui/app/components/sidebar/frame.hbs

```diff
@@ -1,18 +1,16 @@
-<Hds::AppFrame @hasSidebar={{@showSidebar}} as |Frame|>
+<Hds::AppFrame @hasSidebar={{@showSidebar}} @hasHeader={{false}} @hasFooter={{false}} as |Frame|>
   <Frame.Sidebar data-test-sidebar-nav>
     <Hds::SideNav @isResponsive={{true}} @hasA11yRefocus={{true}} @a11yRefocusSkipTo="app-main-content">
       <:header>
         <Hds::SideNav::Header>
           <:logo>
-            {{#if this.currentCluster.cluster}}
-              <Hds::SideNav::Header::HomeLink
-                @icon="vault"
-                @route="vault.cluster"
-                @model={{this.currentCluster.cluster.name}}
-                @ariaLabel="home link"
-                data-test-sidebar-logo
-              />
-            {{/if}}
+            <Hds::SideNav::Header::HomeLink
+              @icon="vault"
+              @route="vault.cluster"
+              @model={{this.currentCluster.cluster.name}}
+              @ariaLabel="home link"
+              data-test-sidebar-logo
+            />
```

Comment:
This conditional was originally added for an earlier iteration of the nav addon since it seemed to render the link before the cluster was loaded which resulted in an error. This no longer seems to be an issue with the HDS version of the addon.